rust/generate_message/tests/unwasm.rs

```diff
@@ -0,0 +1,21 @@
+pub mod common;
+use crate::common::{assert_cmd_stdout, assert_files_eq, setup, teardown};
+
+use std::path::PathBuf;
+
+#[test]
+fn it_unwasm() {
+    let files_dir = PathBuf::from("./tests/it_unwasm");
```

Comment:
_probably_ should make all these relative variables as absolute with `env!(CARGO_MANIFEST_DIR).push("./tests/it_unwasm")` but I cannot really come up with a counterexample to the relative dir off the top of my head.